README.md

```diff
@@ -26,7 +28,7 @@ Download and install [Python](https://www.python.org/downloads/)
 
 Download or [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repository
```

Comment:
Technically you don't need to do this anymore! `pip install dacsspace` will install everything you need. The only reason to pull down the repository is if you want to install a version that has not yet been pushed to PyPI.